camel/loaders/chunkr_reader.py

```diff
@@ -0,0 +1,152 @@
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+import json
+import logging
+import os
+import time
+from typing import IO, Any, Optional, Union
+
+import requests
+
+from camel.utils import api_keys_required
+
+CHUNKR_ENDPOINT = "https://api.chunkr.ai/api/v1/task"
```

Comment:
It would be better to set this as one arg during initialization, as users may choose to deploy the service themselves